README.md

```diff
@@ -1,17 +1,31 @@
 ![MuShop Logo](./images/logo.png)
 
 ---
-
-This is a Terraform configuration that deploys the MuShop basic sample application on [Oracle Cloud Infrastructure (OCI)][oci]. 
-
-MuShop basic is a 3-tier web application that implements an e-commerce site. It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci]. This application is designed to run using only the Always Free tier resources. 
-The repository contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack, 
-that creates all the required resources and configures the application on the created resources.
+MuShop is a showcase of several [Oracle Cloud Infrastructure (OCI)][oci] services in a unified reference application. The sample application implements an e-commerce platform built as a set of micro-services. The accompanying content can be used as a self paced workshop to get started with cloud native application development on [Oracle Cloud Infrastructure (OCI)][oci].
```

Comment:
I think we should remove ~as a self paced workshop~ here, as that is not clearly presented elsewhere.